instrumentation/kafka/kafka-clients/kafka-clients-2.6/library/README.md

```diff
@@ -72,12 +72,32 @@ Then use the `tracingConsumer` as usual for receiving messages from the Kafka cl
 ### Usage (Metrics)
 
 The Kafka client exposes metrics via `org.apache.kafka.common.metrics.MetricsReporter` interface.
-OpenTelemetry provides an implementation that bridges the metrics into OpenTelemetry.
+OpenTelemetry provides an implementation that bridges the metrics into OpenTelemetry. There are two
```

Comment:
This project only deals with manual instrumentation and is not needed for jmx. The jmx module already describes how to collect metrics for a variety of libraries and offers predefined rules, perhaps these should be augmented to include kafka client metrics?